Makefile

```diff
@@ -93,8 +93,8 @@ gen-swagger-apple-chip-cpu:
 		--api-package models \
 		--git-user-id ${GIT_ORG} \
 		--git-repo-id ${GIT_REPO} \
-		-o ./${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
-		-i ./${SPEC_PATCHED_FILE}
+		-o /local/${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
```

Comment:
This task looks like it's running locally, so the `/local` directory may not exist or have the expected contents.  I don't think we have a task like this in `Makefiles` for other generated SDKs; do we need it here for some reason or can we standardize on running the generator in a container?

Reply:
Good call out! I actually was a little confused about the `/local` directory but your comment clears it up. That explains an error I was getting in the GHA Workflow with the `gen-swagger` job when I used the `./` path instead. I blindly updated all because I was working on a few things at once but I'll get it cleared up.

This is necessary for the new OS X Apple Silicon M1/M2 CPU. They can't run the `swagger-codegen` Docker Container because the architecture doesn't match. Results in a hanging container that never continues running. I have the M2 Chip and the only way to use swagger codegen on my local machine is to use the jar directly. I'll add the local Docker run to get the output for you and add it to this discussion.

Reply:
```
docker run --rm -u 502:20 -v /Users/thogarty/workspace/fabric-go:/local swaggerapi/swagger-codegen-cli-v3:3.0.34 generate -l go \
		--additional-properties packageName=v4 \
		--model-package types \
		--api-package models \
		--git-user-id equinix-labs \
		--git-repo-id fabric-go \
		-o /local/fabric/v4 \
		-i /local/spec.patched.yaml
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

Even when requesting the specific platform it isn't a fixing change because they don't support `linux/arm64/v8` AFAICR from the research I performed while debugging this the first time around.

Reply:
The 3.0.34 jar will produce the same Fabric-Go SDK that is currently present in the repo that was generated with the container method. Aside from 2 go packages that were updated and released on Sept. 5th, 2023. Those are being upgraded in the `go.mod` and `go.sum` files. Which is actually the reason that the GHA Workflow for this PR is failing now as well.

Reply:
To clarify your earlier comment, you got an error from the generator when passing in `--platform linux/amd64` to force docker to use the platform that the image supports?  When I run `docker run --platform linux/amd64 swaggerapi/swagger-codegen-cli` on my machine (M1), it outputs the help text for the Swagger CLI, but I haven't gone beyond that to see what happens if I try generating the code.

Reply:
I have tried it. It results in an error specifying that the docker image doesn't support the platform given. Sorry that I missed this comment!

Reply:
OK, I think I managed to reproduce the issue locally.

In my Docker settings, under `Features in development`, I had `Use Rosetta for x86/amd64 emulation on Apple Silicon` enabled.  With that setting enabled, I still get platform warnings but the `make generate` command runs to completion and any performance hit is negligible.

When I disable that setting, the `make generate` command hangs but doesn't exit; it's possible if I left it long enough it would finish, but it's slow enough to be unusable.

Could you try enabling the `Use Rosetta for x86/amd64 emulation on Apple Silicon` setting and see if that works for you as well?  My concerns with introducing an architecture-specific task are that (1) it increases the distance between what is run locally and what is run in CI, making issues harder to reproduce and (2) it increases the chance for mistakes, since anyone running on Apple silicon has to remember to manually run the same tasks, in the same order, that others would run via `make generate`.

Reply:
That is a great find, @ctreatma ! Thank you for sharing. I'll update this PR to match. Worked just fine 👍